### ISSUE TITLE
Won't run whatsoever under Sandboxie

### ISSUE THREAD

**Issue by Beligertint** [ipfs/go-ipfs#2436]
On Windows, trying to run this under Sandboxie does nothing. From the command line, the ipfs command in the folder containing ipfs does nothing. It seems to run for a split second then quietly fails. It runs fine outside of the Sandbox, but instead, it doesn't even list commands when run with no options.

**Comment by djdv** [comment]
I tried this out with Sandboxie 5.11.3, I launched a shell from it and then issued commands to ipfs. Everything seemed to work fine and remain isolated as well.
![screenshot](https://ipfs.pics/ipfs/QmPEvw6zAH9iMRZykJno9hUv5zikckjedZQr9SxdLGgzeR)

**Comment by whyrusleeping** [comment]
whoa. thats a lot of addresses that ipfs is listening on...

**Comment by djdv** [comment]
@whyrusleeping 
Most of those are virtual interfaces, there's only 2 physical and only one of those is active. Some of the addresses ipfs is listening on seem invalid.
I don't have anything assigned to 169.* to my knowledge, only these should be assigned. Physical and virtual interfaces grouped here.
```
[p]192.168.1.40
[p]fd42:562a:6d05::3

[v]192.168.148.1
[v]fd42:562a:6d05::7

[v]192.168.159.1
[v]fd42:562a:6d05::11

[v]2001:0:9d38:6ab8:4b3:1388:9d8d:1bb5
[v]fd42:562a:6d05::22
```

If I recall correctly it wasn't always listening on those but I'm unsure of when that started happening, it may have been when I changed the port and added utp, maybe after I installed cjdns. I'm not sure if the problem is in IPFS or my setup.
